src/__tests__/Home.js

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import '@testing-library/jest-dom';
+
+import Home from '../pages/Home';
+import VideoCard from '../components/VideoCard';
+
+beforeEach(() => {
+    render(<Home />);
+});
+
+describe('<Home />', () => {
+
+    test('Home works', () => {
+        const titleSection = screen.getByTestId('page-name');
+
+        expect(titleSection).toHaveTextContent('Welcome to the Challenge!');
+        expect(titleSection).not.toHaveTextContent('feed');
+    });
+
+    test('Should load all cards', () => {
+        const cardItems = screen.queryAllByTestId('video-card-item');
+
+        for (let i = 0; i < cardItems.length; i++) {
+            expect(cardItems[i]).toBeInTheDocument();
+        }
```

Comment:
This test case may be a little redundant since it is asserting the same elements its receiving from the query (i.e. if one or every element is missing in the render the test would still pass) so it could be better to assert the length:
```javascript
expect(cardItems.length).toBeGreaterThan(0);

// Even better if you are using mock data where you already know the exact expected length
expect(cardItems.length).toBe(10)
```